app/views/paloma/_hook.html.erb

```diff
@@ -2,18 +2,20 @@
 
 <div class="js-paloma-hook" data-id="<%= id %>">
   <script type="text/javascript">
-    (function(){
-      Paloma.env = '<%= Rails.env %>';
+      if (typeof Paloma !== 'undefined') {
```

Comment:
@bcavileer Can we just simplify this by using `if (Paloma == null){ return; }` inside the function? So that we can eliminate nesting. Thanks.